stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/controller/QueryChannelsControllerImpl.kt

```diff
@@ -192,13 +192,8 @@ internal class QueryChannelsControllerImpl(
                 _endOfChannels.value = true
             }
             // first things first, store the configs
-            val configEntities = channelsResponse.associateBy { it.type }.values.map {
```

Comment:
Was this `associateBy` not needed here, are there no duplicate types in this response?

Reply:
If there were any doubling we had to remove it before. When obtaining results. How do you can rely on an answer from BE and predict your app behaviour when you obtain data with doubling and store it in your DB?

Reply:
Changed collection type to Set